src/platforms/node/guides/aws-lambda/layer/index.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: AWS Lambda Layer (Node)
+description: "Learn how to set up Sentry with a Layer"
+---
+
+An alternative way of installing Sentry is using a [Lambda Layer](https://docs.aws.amazon.com/lambda/latest/dg/configuration-layers.html). This happens instead of adding `@sentry/serverless` with NPM and Yarn. Your can still import Sentry as usual:
```

Comment:
```suggestion
You can also install Sentry using a [Lambda Layer](https://docs.aws.amazon.com/lambda/latest/dg/configuration-layers.html) instead of adding `@sentry/serverless` with `npm` or `yarn`. Import Sentry as usual:
```